spmd/compiler/distribute.py

```diff
@@ -603,7 +603,7 @@ def distribute(
     dist_graph: DistributedGraph,
     param_schema: Schema,
     input_schemas: Sequence[Placement],
-    force_compile: bool,
+    optimize_first_iter: bool,
```

Comment:
I believe this will break fbcode stuff.

Reply:
This is reverting a change that was made in a previous PR by mistake I think. We are only using `optimize_first_iter` everywhere.